dev-docs/modules/gdprEnforcement.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: page_v2
+page_type: module
+title: GDPR Enforcement Module
+description: Module to enforce GDPR consent
+module_code : gdprEnforcement
+display_name : GDPR Enforcement
+enable_download : true
+sidebarType : 1
+---
+
+# GDPR Enforcement Module
+{: .no_toc }
+
+* TOC
+{: toc }
+
+{: .alert.alert-warning :}
+This module requires the [EU GDPR consent management module](/dev-docs/modules/consentManagement.html), which reads consent values from the Consent Management Platform (CMP). The GDPR enforcement module
+will then enforce the results. See the base module page for general background, usage, and legal disclaimers.
+
+## Overview
+
+The base [EU GDPR consent management module](/dev-docs/modules/consentManagement.html) performs the following actions:
+
+1. Fetch the user's GDPR consent data from the CMP.
+2. Incorporate this data into the auction objects for adapters to collect.
+
+This GDPR enforcement module adds the following:
+
+3. Allows the page to define which activities should be enforced at the Prebid.js level.
+4. Actively enforces those activities based on user consent data.
+
+The following table details the Prebid.js activities that fall under the [Transparency and Consent Framework (TCF)](https://iabeurope.eu/iab-europe-transparency-consent-framework-policies/) scope:
+
+{: .table .table-bordered .table-striped }
+| TCF Purpose | In-Scope Activity | Enforcement Activity | Optional Controls |
+| --- | --- | --- | --- |
+| Purpose 1 - Store and/or access information on a device | usersync pixels | May result in preventing one or more usersync activities for one or more vendors. | Do not enforce Purpose 1. Do not enforce Purpose 1 vendor signals. Do not enforce Purpose 1 for vendor V. |
+| Purpose 1 - Store and/or access information on a device | user ID modules | May result in preventing one or more UserID modules to not activate. | Do not enforce Purpose 1. Do not enforce Purpose 1 vendor signals. Do not enforce Purpose 1 for vendor V. |
```

Comment:
Can we get rid of the double negative here (prevent...to not)? Either "May prevent one or more UserID modules from activating" or "May allow one or more UserID modules to activate". (I wasn't sure which was correct.)